packages/driver/src/dom/visibility.js

```diff
@@ -430,6 +483,10 @@ const getReasonIsHidden = function ($el) {
     return `This element '${node}' is not visible because it has an effective width and height of: '${width} x ${height}' pixels.`
   }
 
+  if (elIsHiddenByTransfrom($el)) {
+    return `This element '${node}' is not visible because it is hidden by transform.`
```

Comment:
Is there any more information we can provide the user in this error message? Maybe include the value of the transform?

Reply:
Unfortunately, that is impossible. When you fetch transform value with `jquery.css()` or `getComputedStyle()`, the returned value is in the form of `matrix(a, b, c, d, e, f, g)`. 

We cannot get the original string of the transform.